_docs/design-by-contract.md

```diff
@@ -205,15 +205,15 @@ We do offer two more variables which are available to our constraints, additiona
 Design by Contract is fully integrated into the appserver.io infrastructure's autoloading capabilities. This is due to the 
 heavy modifications it requires for structure definitions and the applied caching in the file system.
 
-> To change any Design by Contract behaviour the change therefore has to be made within the class loader configuration
+> To change any Design by Contract behavior, the change must be made within the class loader configuration.
 
-The class loader configuration can be found within any `context.xml` file relevant to the application changes should apply to.
+The class loader configuration can be found within any `context.xml` file, and are only relevant to the part of the application the changes should be applied.
```

Comment:
This should state that if you want to change a certain applications behavior you have to change the `context.xml` file of this specific application. A `context.xml` always affects the complete application or even all applications globally, not just a part of an application.